src/features/counterfactual/CounterfactualForm.tsx

```diff
@@ -113,6 +114,30 @@ export const CounterfactualForm = ({
   return (
     <>
       <form onSubmit={handleSubmit}>
+        <Alert severity="info" sx={{ mb: 2 }}>
+          Executing this transaction will also activate your account. Additional network fees will apply. Base fee is{' '}
+          <strong>
+            {getTotalFeeFormatted(
+              advancedParams.maxFeePerGas,
+              advancedParams.maxPriorityFeePerGas,
+              BigInt(gasLimit?.baseGas || '0') + BigInt(gasLimit?.safeTxGas || '0'),
+              chain,
+            )}{' '}
+            {chain?.nativeCurrency.symbol}
+          </strong>
+          , activation fee is{' '}
```

Comment:
Should we add here that this fee will only have to be paid once or that future transactions will not have to pay this?

Reply:
Good idea, I reworded it to say "one time activation fee is ..."